### ISSUE TITLE
Going to Keyring shows "Keyring is not saving to disk. Changes will be lost."

### ISSUE THREAD

**Issue by Hew-ux** [NickvisionApps/Parabolic#909]
**Issues:** Opening the program and navigating to "Keyrings" shows the following error message:

"Keyring is not saving to disk. Changes will be lost."

![image](https://github.com/user-attachments/assets/1298b54e-3c69-4b9f-945a-01ea6316629f)

Attempting to make a key will show that this error message is true. Changes are lost upon closing the application.

**Diagnostics:** Running the flatpak via the command line `flatpak run --branch=stable --arch=x86_64 --command=/app/lib/org.nickvision.tubeconverter/org.nickvision.tubeconverter.gnome org.nickvision.tubeconverter` and then navigating to the keyrings page show this in the output:

```bash
(org.nickvision.tubeconverter.gnome:2): Gsk-WARNING **: 15:40:43.846: Clipping is broken, everything is clipped, but we didn't early-exit.
```

In flatpak permissions, turning on "Access to all user files" and "access to all system files" to read/write seems to have no effect on this error.

**System info:** Running Nobara Linux (based on Fedora) via flatpak installation. x86_64.

**Comment by nlogozzo** [comment]
Did you previously use the Keyring feature in old versions of Parabolic?

**Comment by Hew-ux** [comment]
No. In fact, I was attempting to use the keyring for the first time.

**Comment by nlogozzo** [comment]
Hi @Hew-ux are you still experiencing this issue with the latest version of Parabolic?

It seems like there is something misconfigured with your system's secret storage. 

Could you also try installing this app: https://flathub.org/apps/app.drey.KeyRack

And see if you are able to access your system's keyring.

**Comment by nlogozzo** [comment]
Please reopen if there are any updates.

**Comment by Hew-ux** [comment]
@nlogozzo I have updated the flatpak (now on 2025.1.4) and tested this again and now there are no error messages. Credentials are saved successfully between sessions. I did not have to install KeyRack.
The only other thing that has changed is I've upgraded from Nobara Linux 40 to 41, so potentially the OS itself has received a fix.
